pkg/shell/shell-modals.jsx

```diff
@@ -133,11 +134,29 @@ export const LangModal = () => {
                     <Divider />
                     <MenuContent>
                         <MenuList>
-                            {Object.keys(manifest.locales || { })
-                                    .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()))
-                                    .map(key => {
-                                        return <MenuItem itemId={key} key={key} data-value={key}>{manifest.locales[key]}</MenuItem>;
-                                    })}
+                            {
+                                (() => {
+                                    const filteredLocales = Object.keys(manifest.locales || {})
+                                            .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()));
+
+                                    if (filteredLocales.length === 0) {
+                                        return (
+                                            <MenuItem>
```

Comment:
No need to use EmptyState. Just put the translated text inside the MenuItem directly. 
```
<MenuItem>
{_("No results found")}
</MenuItem>
```

https://www.patternfly.org/v4/components/menu#filtering-with-search-input